modules/platforms/cpp/ignite/network/detail/win/sockets.cpp

```diff
@@ -82,10 +83,10 @@ void try_set_socket_options(SOCKET socket, int buf_size, BOOL no_delay, BOOL out
 
     int res = setsockopt(socket, SOL_SOCKET, SO_KEEPALIVE, reinterpret_cast<char *>(&keep_alive), sizeof(keep_alive));
 
-    // TODO: IGNITE-17606 Disable keep-alive once heartbeats are implemented.
+    // TODO: Disable keep-alive once heartbeats are implemented.
```

Comment:
Is there a ticket?

Reply:
Yeah, and the ticket was actually correct. Seems like I messed up something here. Reverted changes.